pymc3/tests/test_distributions.py

```diff
@@ -807,7 +885,8 @@ def test_chi_squared(self):
             lambda value, nu: sp.chi2.logpdf(value, df=nu),
         )
 
-    @pytest.mark.xfail(reason="Poor CDF in SciPy. See scipy/scipy#869 for details.")
+    # TODO: Is this still needed? It passes locally.
+    # @pytest.mark.xfail(reason="Poor CDF in SciPy. See scipy/scipy#869 for details.")
```

Comment:
Usually, an XFAIL should error if the test does NOT fail.
In any case, these commented lines must be dealt with before merging.

Reply:
I probably misinterpreted something somewhere, but this test was not failing (nor failing because it was not failing) on my machine. Anyway, it should soon be cleared up when the tests conclude in here.